huaweicloud/resource_huaweicloud_compute_volume_attach.go

```diff
@@ -6,8 +6,9 @@ import (
 	"time"
 
 	"github.com/chnsz/golangsdk"
-	"github.com/chnsz/golangsdk/openstack/compute/v2/extensions/volumeattach"
```

Comment:
can we totally replace this package?

Reply:
all volumeattach packages have been removed.